### PR TITLE
Frontend/fix: #1017 Unification bugfixes

### PR DESCRIPTION
**Pull request by Rohit4997**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: Frontend/ui-driver/src/Resource/Config.purs [hunk 1]
```diff
@@ -90,15 +90,15 @@ getFingerPrint dummy = do
...
 
 getSupportNumber :: String -> String
-getSupportNumber _ = case getEnv of
-                        DEV  -> ""
-                        UAT  -> ""
-                        PROD -> ""
+getSupportNumber _ = case getEnv of 
+                        DEV  -> "08068501081"
+                        UAT  -> "08068501081"
+                        PROD -> "08068501081"
```
Comment (ashishsingh101) [inline Frontend/ui-driver/src/Resource/Config.purs]: do we need these numbers ?

File: Frontend/ui-customer/src/Services/Config.purs [hunk 2]
```diff
@@ -41,11 +41,14 @@ getEnv = case spy "Selected Environment :- " (environment "") of
...
 
+getMerchantId :: String
+getMerchantId = "NA"
+
 getConfig :: Config
 getConfig = do
   case getEnv of
     LOCAL -> Config
-        { baseUrl: "http://localhost:8013/v2"
+        { baseUrl: ""
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Services/Config.purs]: it should point to local host only for frontend to work with backend

File: Frontend/ui-customer/src/Services/Config.purs [hunk 3]
```diff
@@ -61,44 +64,40 @@ getConfig = do
         , fingerprint : ""
         }
 
-getMerchantId :: String
-getMerchantId = "NA"
 getEndpoint :: String -> String
 getEndpoint dummy = do
-  if ((getKeyInSharedPrefKeysConfig "MOBILE_NUMBER") == "5000500050") then
+  if ((getKeyInSharedPrefKeysConfig "MOBILE_NUMBER_KEY") == "" || (getKeyInSharedPrefKeysConfig "MOBILE_NUMBER_KEY") == "") then
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Services/Config.purs]: both the checks are same only

File: Frontend/ui-customer/src/Components/EmergencyHelp/View.purs [hunk 4]
```diff
@@ -237,15 +241,17 @@ callEmergencyContactConfig state  =
   popUpConfig' = config' {
     primaryText { 
       text = (<>) (getString CALL_EMERGENCY_CONTACTS) "?"
-    , margin = (Margin 40 23 40 12)
+    -- , margin = (Margin 40 23 40 12)
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Components/EmergencyHelp/View.purs]: remove commented lines, other places as well

File: Frontend/ui-common/src/Components/MenuButton/Controller.purs [hunk 5]
```diff
@@ -112,8 +112,8 @@ config =
           , imageUrl : "ny_ic_radio_button,https://assets.juspay.in/nammayatri/images/common/ny_ic_radio_button.png"
           , imageMargin : (Margin 0 0 0 0)
           , imagePadding : (Padding 0 0 0 0)
-          , activeStroke :("2," <> Color.black800)
-          , inActiveStroke :("2," <> Color.black600)
+          , activeStroke :("3," <> Color.black800)
+          , inActiveStroke :("2,#A7A7A7")
```
Comment (ashishsingh101) [inline Frontend/ui-common/src/Components/MenuButton/Controller.purs]: define this in `Color.purs`

File: Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java [hunk 6]
```diff
@@ -3113,13 +3116,14 @@ public void shareTextMessage(String title, String message) {
...
-                        sendIntent.setClipData(clipData);
-                    }
-                    sendIntent.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
                     sendIntent.setType("text/plain");
+                    // if (thumbnailBitmap != null) {
+                    //     Uri thumbnailUri = getImageUri(context, thumbnailBitmap);
+                    //     ClipData clipData = ClipData.newUri(context.getContentResolver(), "Thumbnail Image", thumbnailUri);
+                    //     sendIntent.setClipData(clipData);
+                    //     sendIntent.setType("image/*");
+                    // }
```
Comment (ashishsingh101) [inline Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java]: remove if not needed or else add comment why needed

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

**Review by kavya-shree-s — APPROVED** [review]

**Review by kavya-shree-s — APPROVED** [review]

**Review by ashishsingh101 — APPROVED** [review]

**Pull request merged**